apps/heft/src/schemas/heft.schema.json

```diff
@@ -108,6 +108,34 @@
       "items": { "$ref": "#/definitions/heft-plugin" }
     },
 
+    "aliasesByName": {
+      "type": "object",
+      "description": "Defines aliases for existing Heft actions, and allows them to be invoked by name with default parameters.",
+      "additionalProperties": false,
+      "patternProperties": {
+        "^[a-z][a-z0-9]*([-][a-z0-9]+)*$": {
+          "description": "Defines a Heft action alias.",
+          "type": "object",
+          "additionalProperties": false,
+          "required": ["actionName"],
+          "properties": {
+            "actionName": {
+              "description": "The name of the Heft action to invoke.",
+              "type": "string",
+              "pattern": "^[a-z][a-z0-9]*([-][a-z0-9]+)*$"
+            },
+            "defaultArguments": {
+              "description": "Arguments to pass to the Heft action by default.",
```

Comment:
What does "by default" mean?  If additional parameters are specified, are they blindly appended after the default ones?  (Even if the default ones included `--` or a parameter that cannot be specified multiple times?)

Reply:
"by default" means they're appended after the action name. They are then validated by argparse just like any other arg. This allows for nice things like aliasing `heft run --to x --` so that you don't need to specify all that boilerplate, and instead can just hop to specifying the post-`--` args in the alias command

Reply:
Could you mention that in the description?

Reply:
Done!